Felienne/hedy
# [FIX] Removes programming experience from profile update page

**Description**
In this PR we perform some minor fixes that built upon the work done in #2066. We remove the programming experience section from the profile update page as we are only interested in that data on signup. We also improve the keyword language selector to only show English and the current language (if supported). Otherwise it is automatically hidden and English is selected. Lastly we update the reset password section as we forgot than in #2066.

**Fixes**
This PR fixes #2076.

**How to test**
Navigate to the profile page, verify that the programming experience fields are gone and also verify that everything is still working as expected. Also verify that the forgotten password UI is updated to match other forms.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
